### PR TITLE
Volvo day 5 beta test and misc edits

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ Arcueid. You said that you were rich, right?"
 [sha:70d82c6398f687f2ee46ce1f39f64f39bda16f03]{
 -- Page 130, Offset 8898.
 -- 「現金でもカードでも、とにかく貸してくれ。手当てに必要な物を買いこんでくる。
-"Cash, credit card, whatever you have just lend it to me. I'm going to go buy some first aid.
+"Cash, credit card, or whatever you have, just lend it to me. I'm going to go buy some first aid.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:1136]: revert "or".

File: script/Arcueid/Day 5/05_00_ARC05_4.txt [hunk 2]
```diff
@@ -198,7 +198,7 @@ Just thinking about it makes my limbs tremble.
 [sha:9510d3e8af814e27190d00f8d7bba982badb129c]{
 -- Page 329, Offset 9435.
 -- ナイフを持つ指先が、痛いほど柄を握りしめる。
-I'm gripping my knife so hard it's making my fingers hurt.
+I'm gripping my knife so hard, it's making my fingers hurt.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_4.txt:201]: "so hard that it's" sounds better imo

File: script/Arcueid/Day 5/05_00_ARC05_6_2.txt [hunk 3]
```diff
@@ -374,6 +374,7 @@ Vlov's flames seem to mean nothing to her in that black and white combat dress.
 [sha:d5fb3c5bed963689df7634ee1428a21ba6ebaf15]{
 -- Page 488, Offset 9896.
 -- あの衣装が炎を呑むのか、それとも、体を作り替えた結果あの姿になるだけなのか。
+// too many now's or is that just fine?
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_6_2.txt:377]: meh its fine

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 4]
```diff
@@ -356,6 +356,7 @@ For me, blood is the very essence of warmth.
 [sha:3d75bd2ed77a94faa858d9919f02586581f25b1b]{
 -- Page 541, Offset 10050.
 -- 生命の熱さ。我が凍傷を癒やす霊薬。我が世界、我が原理を回すもの。
+// is the amount of semicolons in this section fine?
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:359]: yeah it's fine

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 5]
```diff
@@ -164,7 +164,7 @@ The lance is readied.
 [sha:a62cb27a6f3af2efe6123eac44f38bbe03a886fa]{
 -- Page 668, Offset 10450.
 -- 本来ならこの大波ごと、その後ろに身を隠す敵を砕くべきだと理解している。
-He knows that by all rights he ought to crush this wave along with the enemies concealed behind it.
+He knows that by all rights, he ought to crush this wave along with the enemies concealed behind it.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:167]: that, by all rights, he ought

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 6]
```diff
@@ -1025,7 +1025,7 @@ If I do not burn it to the ground, it would go against everything that I believe
 [sha:1c569a09df5b70346eb15174ccaa0b9e0ed30421]{
 -- Page 870, Offset 7977.
 -- 真祖の姫。その心臓を、おれに<呉|よ>こせ”」
-Princess of the True Ancestors. That heart of yours, shall be mine."
+%{i}Princess of the True Ancestors%{/i}. That heart of yours, shall be mine."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:1028]: Not sure I'd italicize this one either. It just doesn't feel like a nickname to me, only a description

File: script/Arcueid/Day 5/05_00_ARC05_10B.txt [hunk 7]
```diff
@@ -165,7 +165,7 @@ So... I'm supposed to do this alone?
 -- Page 768, Offset 10741.
 -- 「まあ、動かない標的に引き金を引くだけのお仕事ですし。精神的な負担を無視すれば、の話ですけどね？」
 "Well, I guess all I gotta do is pull the trigger on some stationary targets.
- Long as I completely ignore the emotional toll it's gonna take on me, that is."
+ As long as I completely ignore the emotional toll it's gonna take on me, that is."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_10B.txt:168]: Casual speech, might revert

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt (hunk 1 above)
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:1136]: No `or`

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 8]
```diff
@@ -207,7 +207,8 @@ It would be futile to hold him down directly. Besides, I've never done it before
 [sha:7371bf30d3635da3426e39418d57415e9e5c5622]{
 -- Page 262, Offset 9241.
 -- 「鎖……？　鎖って、あの鎖か？」
-Chains? You mean like %{i}metal%{/i} chains?
+// can the lint be set to detect if both quotes are missing in voiced lines?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:210]: It's pretty dumb rn, I can probably make it smarter by examining the JP line for quote markers

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 9]
```diff
@@ -6,7 +6,7 @@
 [sha:b9ffb48a4d1a0dcdceeaf6c4a2b6a2d99cfdb51b]{
 -- Page 10007, Offset 9202.
 -- お腹減ってるでしょ、志貴？　さっきのおかわり、いる？　新しいの作ってあげよっか？」
-You must be hungry, right Shiki?
+You must be hungry, right, Shiki?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:9]: Not sure about this one, I don't really feel like I pause here when I say this line

File: script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 10]
```diff
@@ -179,12 +179,12 @@ Wait, she just said something completely outrageous―――!
...
 -- Page 383, Offset 9600.
 -- 「う、」
-"%{i}Wah%{/i}."
+"%{i}Wah%{/i}――"
 }
 [sha:be31f004f5600d4de80d1bb0838a5ba7a275e37e]{
 -- Page 384, Offset 9601.
 -- 「うう、」
-"%{i}Waah%{/i}."
+"%{i}Waah%{/i}――"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_5.txt:187]: Maybe have this be 2 CJK dashes so they lengthen with the waah. Or just have them all be 3

File: script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 11]
```diff
@@ -266,7 +266,7 @@ We've landed in a hanging garden; one that was not originally designed for human
 [sha:0b586228f693fcee8ff039f4e779cece7edeedbf]{
 -- Page 393, Offset 9617.
 -- 「っ―――おまえ、な、なんて無茶を―――」
-"Huh―――you, w-what the hell was that about―――"
+"Eh―――you, w-what the hell was that about―――"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_5.txt:269]: Why bring back eh?

File: script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 12]
```diff
@@ -205,7 +205,7 @@ No, we're jumping.
 -- Page 387, Offset 9605.
 -- アルクェイドはビルの壁にジャンプし、張り付いてからさらに跳躍、反転を繰り返して、あっという間にビルの屋上……いや、上空まで舞い上がった。
 Arcueid bounds from wall to wall up the face of the opposing buildings.
- Before I even have time to blink we're on top of the building―――no, soaring across the sky.
+ Before I even have time to blink, we're on top of the building―――no, soaring across the sky.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_5.txt:208]: Not sure I like this pause

File: script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 13]
```diff
@@ -648,7 +648,7 @@ The wrath of nature that crushes it.
 [sha:0cbd0216accdb72d6cea8f400853a48dee602b4c]{
 -- Page 419, Offset 9693.
 -- その体に備わった当然の権利である。
-Of course such a body would possess these rights.
+Of course, such a body would possess these rights.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_5.txt:651]: Revert. Of course with a comma feels like a you are saying something that is counter-flow to the rest of the conversation, but this is "of _course_ such a body would possess these rights"

File: script/Arcueid/Day 5/05_00_ARC05_5B.txt [hunk 14]
```diff
@@ -379,5 +379,5 @@ I reach the first floor.
 [sha:b1d0ba364a53c10222c6f4f749883ace7ee5c4e3]{
 -- Page 447, Offset 9776.
 -- 俺はナイフを握り直して、出来たばかりの爆心地に走り出した。
-I adjust my grip on my knife and run toward the newly formed crater.
+I adjust the grip on my knife and run toward the newly formed crater.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_5B.txt:382]: Adjusting _the_ grip on the knife makes it sounds like he's diddling with the makeup of the handle. He's adjusting _his_ hands/action. If you don't like duplicate his I'd change the knife not his grip

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 15]
```diff
@@ -161,7 +161,7 @@ But that glacier formed in an instant.
 [sha:a756d8ecb94b486da90c3b1e0f83c70b75e1d5e1]{
 -- Page 526, Offset 10011.
 -- 範囲は狭く、寒波は地を這うものだが、放たれてしまえば逃げる間もなく殺される。
-Its range is narrow, and the cold wave only crawls along the ground, but once it's released there would be no hope of escaping it.
+Its range is narrow, and the cold wave only crawls along the ground, but once it's released, there would be no hope of escaping it.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:164]: Feels like too many pauses to me

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt (hunk 5 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:167]: Reads better to me with no commas tbh

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_9_2B.txt [hunk 16]
```diff
@@ -273,7 +273,7 @@ A human cannot pass through a shower of debris on his own.
 [sha:f6dc95af6904944bd4332a85c1d88744049ea3db]{
 -- Page 674, Offset 10471.
 -- 真祖の姫による援護を満足に活用する事もできない。
-He cannot even make use of support from the Princess of the True Ancestors to that end.
+He cannot even make use of the support from the Princess of the True Ancestors to that end.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_9_2B.txt:276]: Not sure I like the here. Either `make use of support from` or `make use of the support of`

**Pull request merged**